File: add/codesnippet/VisualBasic/m-system.runtime.interop_30_1.vb
        ' Create an instance of an unmanged COM object.
        Dim UnmanagedComClassInstance As New UnmanagedComClass()

        ' Create a string to pass to the COM object.
        Dim helloString As String = "Hello World!"

        ' Wrap the string with the VariantWrapper class.
        Dim var As Object = New System.Runtime.InteropServices.VariantWrapper(helloString)

        ' Pass the wrapped object.
        UnmanagedComClassInstance.MethodWithStringRefParam(var)

File: add/codesnippet/CSharp/m-system.runtime.interop_30_1.cs
        // Create an instance of an unmanged COM object.
        UnmanagedComClass UnmanagedComClassInstance = new UnmanagedComClass();

        // Create a string to pass to the COM object.
        string helloString = "Hello World!";

        // Wrap the string with the VariantWrapper class.
        object var = new System.Runtime.InteropServices.VariantWrapper(helloString);

        // Pass the wrapped object.
        UnmanagedComClassInstance.MethodWithStringRefParam(ref var);